frontend/src/reviews/ReviewDecisionDisplay.tsx

```diff
@@ -69,16 +112,41 @@ export default function ReviewDecisionDisplay({ response, modelId }: ReviewDecis
                 <Typography variant='caption'>as {getRoleDisplay(response.role, modelRoles)}</Typography>
               )}
             </Stack>
-            <Typography fontWeight='bold'>{formatDateString(response.createdAt)}</Typography>
+            <Stack direction='row' alignItems='center' spacing={1}>
+              <Typography fontWeight='bold'>{formatDateString(response.createdAt)}</Typography>
+              <IconButton onClick={(event) => setAnchorEl(event.currentTarget)}>
+                <MenuIcon />
```

Comment:
I can see this is simply following `ReviewCommentDisplay`, but can we use the following icon instead for both components? https://mui.com/material-ui/material-icons/?query=more&selected=MoreHoriz